src/styles/index.scss

```diff
@@ -1,3 +1,88 @@
+$stopwatcDmtr: 80vmin;
+$centerDmtr: 5vmin;
+$secondsW: 1.5vmin;
+$secondsH: 38vmin;
+$minutesW: 3vmin;
+$minutesH: 20vmin;
+
+@mixin animation($duration, $timing) {
+  animation-name: rotate;
+  animation-duration: $duration;
+  animation-direction: normal;
+  animation-iteration-count: infinite;
+  animation-timing-function: $timing;
+}
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+.container {
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
+  position: relative;
+  width: $stopwatcDmtr;
+  height: $stopwatcDmtr;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+
+  &__minutes-hand {
+    position: absolute;
+    height: 20vmin;
```

Comment:
The height value here is hardcoded as `20vmin`, but you have a variable `$minutesH` defined. Consider using `$minutesH` for consistency and maintainability.